ecosystem_tests/dorkl.py

```diff
@@ -124,6 +124,62 @@ def docker_exec(cmd, timeout=TIMEOUT, log=True, detach=False):
             container_name=container_name, cmd=cmd), timeout, log, detach)
 
 
+def replace_file_on_manager(local_file_path, manager_file_path):
+    """ Remove a file and upload a new one.
+
+    :param local_file_path:
+    :param manager_file_path:
+    :return:
+    """
+    if os.path.isdir(local_file_path):
```

Comment:
Looks like you don't need the else and the if should wrap the `rm` command:
if os.path.isdir(...):
docker_exec('rm -rf {destination}'.format(
            destination=manager_file_path))

the rest commands are the same.